hubble/Makefile

```diff
@@ -62,5 +61,20 @@ local-release: clean
 		rm -r release/$$OS; \
 	done;
 
+install: install-binary install-bash-completion-only
+
+install-binary:
+	$(QUIET)$(INSTALL) -m 0755 -d $(DESTDIR)$(BINDIR)
+	$(QUIET)$(INSTALL) -m 0755 $(TARGET) $(DESTDIR)$(BINDIR)
+
+install-bash-completion: $(TARGET) install-bash-completion-only
```

Comment:
`install-bash-completion-only` depends on `$(TARGET)`, so as-is it can fail when invoked directly. If we add `$(TARGET)` as a prerequisite to `install-bash-completion-only` then I'm not sure why we need two bash completion targets.

Here we're relying on make processing prerequisites in declaration order which AFAIK is not always guaranteed depending on the version and/or parallelization level.